src/arc.ts

```diff
@@ -586,6 +599,90 @@ export function pollForAccountChanges(currentAccountAddress: Address | null, int
   });
 }
 
+async function monitorGraphNodeSubgraph() {
+  if (targetedNetwork() === "ganache") {
```

Comment:
why condition that with "ganache"?

Reply:
because checking the subgraph status is done with different queries then checking on graphnode

Reply:
why not making that general via the web3 provider ?
why there is unique functionality for ganache ?

Reply:
how do I check the graph status via the web3 provider?

Reply:
my question was why there is a need to explicitly check for the network type ("ganache") .
the web3 provider already knows that

Reply:
because we do different queries when interacting with ganache or the graph nodde